packages/fxa-content-server/server/lib/flow-event.js

```diff
@@ -202,6 +203,17 @@ function logFlowEvent(event, data, request) {
 
   // The data pipeline listens on stderr.
   process.stderr.write(JSON.stringify(eventData) + '\n');
+  logStatsdPerfEvent(eventData);
+}
+
+function logStatsdPerfEvent(eventData) {
+  if (eventData.event.startsWith('flow.performance.')) {
```

Comment:
I would probably add a link/comment to describe what kind of `flow.performance` events are. Ref https://github.com/mozilla/fxa/blob/77a2c2821af087d5e5ccdabeff7061d440986e10/packages/fxa-auth-server/docs/metrics-events.md#success-event-names

Reply:
Ok. I'll sneak that in in another metrics related PR.  I don't want to run CI again.  😬